### PR TITLE
Fix external ozone

### PR DESCRIPTION
**Pull request by jkim-julie**
It fixes compilation errors with ozone_platform_wayland_external=true. PTAL.

### CODE REVIEW

File: src/ozone/wayland/screen.cc [hunk 1]
```diff
@@ -73,7 +73,6 @@ void WaylandScreen::OutputHandleMode(void* data,
                                      int32_t width,
                                      int32_t height,
                                      int32_t refresh) {
-  WaylandScreen* screen = static_cast<WaylandScreen*>(data);
```
Comment (msisov) [inline src/ozone/wayland/screen.cc:76]: that's gonna crash. Check line 79

Comment (jkim-julie) [reply src/ozone/wayland/screen.cc:76]: There is another definition in the line 78.

File: src/ozone/ui/desktop_aura/desktop_window_tree_host_ozone_wayland.cc [hunk 2]
```diff
@@ -1007,8 +1008,10 @@ void DesktopWindowTreeHostOzone::InitOzoneWindow(
   const gfx::Rect& bounds_in_pixels = ToPixelRect(params.bounds);
   const gfx::Rect& bounds = gfx::Rect(bounds_in_pixels.origin(),
                                       AdjustSize(bounds_in_pixels.size()));
+  ui::PlatformWindowInitProperties properties;
+  properties.bounds = bounds;
   platform_window_ =
-      ui::OzonePlatform::GetInstance()->CreatePlatformWindow(this, bounds);
+      ui::OzonePlatform::GetInstance()->CreatePlatformWindow(this, properties);
```
Comment (msisov) [inline src/ozone/ui/desktop_aura/desktop_window_tree_host_ozone_wayland.cc]: std::move(properties).

File: src/ui/aura/window_tree_host_platform.cc [hunk 3]
```diff
@@ -70,7 +70,7 @@ void WindowTreeHostPlatform::CreateAndSetPlatformWindow(
     ui::PlatformWindowInitProperties properties) {
 #if defined(OZONE_PLATFORM_WAYLAND_EXTERNAL)
   platform_window_ =
-      ui::OzonePlatform::GetInstance()->CreatePlatformWindow(this, properties.bounds);
+      ui::OzonePlatform::GetInstance()->CreatePlatformWindow(this, properties);
```
Comment (msisov) [inline src/ui/aura/window_tree_host_platform.cc]: > #if defined(USE_OZONE) || defined(OZONE_PLATFORM_WAYLAND_EXTERNAL)
>  platform_window_ =
>      ui::OzonePlatform::GetInstance()->CreatePlatformWindow(this, std::move(properties));
> #if defined(OZONE_PLATFORM_WAYLAND_EXTERNAL)
> bool ime_enabled =
>  base::CommandLine::ForCurrentProcess()->HasSwitch(switches::kEnableNevaIme);
> if (ime_enabled)
>  GetInputMethod()->AddObserver(this);
> SetImeEnabled(ime_enabled);
> #endif

File: src/ozone/wayland/screen.cc (hunk 1 above)
Comment (msisov) [reply src/ozone/wayland/screen.cc:76]: oh, indeed

File: src/ozone/ui/desktop_aura/desktop_window_tree_host_ozone_wayland.cc (hunk 2 above)
Comment (jkim-julie) [reply src/ozone/ui/desktop_aura/desktop_window_tree_host_ozone_wayland.cc]: Done.

File: src/ui/aura/window_tree_host_platform.cc (hunk 3 above)
Comment (jkim-julie) [reply src/ui/aura/window_tree_host_platform.cc]: Done.

**Comment by jkim-julie** [comment]
I Updated the branch. PTAL.

**Comment by msisov** [comment]
lgtm

**Pull request merged**